update.sh

```diff
@@ -82,7 +85,7 @@ print_baseimage() {
 	    OPENHAB_URL="$openhab_url" \\
 	    OPENHAB_VERSION="$version"
 
-	EOI
+EOI
```

Comment:
Can you revert the changes on lines 79 and 88? The generated Dockerfiles now contain `\\` and $base_image instead of the variable value.

Reply:
Fixed